lib/blacklight/document_presenter.rb

```diff
@@ -87,7 +87,26 @@ def render_field_value value=nil, field_config=nil
         safe_values = safe_values.map { |x| content_tag :span, x, :itemprop => field_config.itemprop }
       end
 
-      safe_join(safe_values, (field_config.separator if field_config) || field_value_separator)
+      render_values(safe_values, field_config)
+    end
+
+    ##
+    # Render a fields values as a string
+    # @param [Array] values to display
+    # @param [Blacklight::Solr::Configuration::Field] solr field configuration
+    # @return [String]
+    def render_values(values, field_config = nil)
```

Comment:
I'm looking for a better name for this method, but haven't thought up anything yet...